test/AWS.Deploy.Orchestration.UnitTests/DeployedApplicationQueryerTests.cs

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Amazon.CloudFormation;
+using Amazon.CloudFormation.Model;
+using AWS.Deploy.Common;
+using AWS.Deploy.Orchestration.Data;
+using AWS.Deploy.Orchestration.Utilities;
+using AWS.Deploy.Recipes.CDK.Common;
+using Moq;
+using Xunit;
+
+namespace AWS.Deploy.Orchestration.UnitTests
+{
+    public class DeployedApplicationQueryerTests
+    {
+        private readonly Mock<IAWSResourceQueryer> _mockAWSResourceQueryer;
+
+        public DeployedApplicationQueryerTests()
+        {
+            _mockAWSResourceQueryer = new Mock<IAWSResourceQueryer>();
+        }
+
+        [Fact]
+        public async Task GetExistingDeployedApplications_ListDeploymentsCall()
+        {
+            var stack = new Stack {
+                Tags = new List<Tag>() { new Tag {
+                    Key = CloudFormationIdentifierConstants.STACK_TAG,
+                    Value = "AspNetAppEcsFargate"
+                } },
+                Description = CloudFormationIdentifierConstants.STACK_DESCRIPTION_PREFIX,
+                StackStatus = StackStatus.CREATE_COMPLETE,
+                StackName = "Stack1"
+            };
+
+            _mockAWSResourceQueryer
+                .Setup(x => x.GetCloudFormationStacks())
+                .Returns(Task.FromResult(new List<Stack>() { stack }));
+
+            var deployedApplicationQueryer = new DeployedApplicationQueryer(_mockAWSResourceQueryer.Object);
+
+            var result = await deployedApplicationQueryer.GetExistingDeployedApplications();
+            Assert.Single(result);
+
+            var expectedStack = result.First();
+            Assert.Equal("Stack1", expectedStack.StackName);
+        }
+
+        [Fact]
+        public async Task GetExistingDeployedApplications_DeployCall()
+        {
+            var stack = new Stack
+            {
+                Tags = new List<Tag>() { new Tag {
+                    Key = CloudFormationIdentifierConstants.STACK_TAG,
+                    Value = "AspNetAppEcsFargate"
+                } },
+                Description = CloudFormationIdentifierConstants.STACK_DESCRIPTION_PREFIX,
+                StackStatus = StackStatus.CREATE_COMPLETE,
+                StackName = "Stack1"
+            };
+
+            _mockAWSResourceQueryer
+                .Setup(x => x.GetCloudFormationStacks())
+                .Returns(Task.FromResult(new List<Stack>() { stack }));
+
+            var deployedApplicationQueryer = new DeployedApplicationQueryer(_mockAWSResourceQueryer.Object);
+
+            var result = await deployedApplicationQueryer.GetExistingDeployedApplications(new List<Recommendation>());
+            Assert.Empty(result);
+        }
+
+        [Theory]
+        [InlineData("", CloudFormationIdentifierConstants.STACK_DESCRIPTION_PREFIX, "CREATE_COMPLETE")]
```

Comment:
`StackStatus.CREATE_COMPLETE`

Similarly for other InlineData.

Reply:
It is a static member and cannot be used